File: PB - More Exercises JavaScript/1.0 Simple Operations and Calculations - More Exercises/1.0 Simple Operations and Calculations - More Exercises/04. Vegetable Market.js
function vegetableMarket(...arg) {
    let [vegetable, fruit, kgVegetable, kgFruit] = [...arg.map(Number)]
    return ((vegetable * kgVegetable + fruit * kgFruit) / 1.94).toFixed(2)
}

// console.log(vegetableMarket(0.194, 19.4, 10, 10))  //101.00

File: PB - More Exercises GO/1.0 Simple Operations and Calculations - More Exercises/1.0 Simple Operations and Calculations - More Exercises/04. Vegetable Market.go
package main

import "fmt"

func main() {
    var vegetable, fruit, kgVegetable, kgFruit float64
    fmt.Scan(&vegetable, &fruit, &kgVegetable, &kgFruit)
    fmt.Printf("%.2f", (vegetable * kgVegetable + fruit * kgFruit) / 1.94)
}


/*
name   :04. Vegetable Market
input  :0.194 19.4 10 10
output :101.00
*/

